docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,576 +1,641 @@
+| `ScaleDimensions`   | `Vector3`                  | cube<br>octahedron<br>bevelled cube  | The dimensions of the cuboid. Scales in the same way as for a 9-patch image. The default value is `[1, 1, 1]`.     |
+| `BevelPercentage`   | `float` in a range 0-1     | bevelled cube                        | Specifies how bevelled the cuboid should be, based on the smallest dimension. It affects the ratio of the outer face widths to the width of the overall cube. The default value is `0.0` - no bevel. |
+| `BevelSmoothness`   | `float` in a range 0-1     | bevelled cube                        | Specifies how smooth the bevelled edges should be. The default value is `0.0` - sharp edges.                       |
+| `LightPosition`     | `Vector3`                  | all                                  | The absolute position of the light source - the point from which the object is illuminated. All zeros indicate the top-left corner in the visual plane. |
 
-**Table: Mesh visual properties**
+There are seven `Shape` values, some of which are specializations as compared to another:
 
-| Property         | Type              | Required          | Description                              |
-|----------------|-----------------|-----------------|----------------------------------------|
-| `ObjectURL`      | `string`            | Yes               | The location of the `.obj` file.         |
-| `MaterialURL`    | `string`            | No                | The location of the `.mtl` file. Leave blank for a textureless object. |
-| `TexturesPath`   | `string`            | If using material | The path to the directory where the textures (including gloss and normal) are stored. |
-| `ShadingMode`    | `integer` or `string` | No                | The type of the shading mode that the mesh uses. |
-| `UseMipmapping`  | `boolean`           | No                | Whether to use mipmaps for textures. By default, `true`. |
-| `UseSoftNormals` | `boolean`          | No                | Whether to average normals at each point to smoother the textures. By default it is `true`. |
-| `LightPosition`  | `Vector3`           | No                | The position, in the stage space of the point light that applies lighting to the model. |
+<a name="table_primit_possible_values"></a>
+**Table: Shape values**
 
--   The `ShadingMode` property defines the shading mode type used by the mesh.
+| Enumeration value | Description                                                                      |
+|-------------------|----------------------------------------------------------------------------------|
+| `Sphere`          | The default shape.                                                               |
+| `ConicalFrustrum` | The area bound between two circles - a cone with the tip removed.                |
+| `Cone`            | Equivalent to a conical frustrum with a top radius of 0.                         |
```

Comment:
Global: I think you need to verify the spelling of "frustrum" , it is actually "frustum". 

https://www.google.com/search?client=firefox-b-d&q=frustrum

Reply:
yes, I know :)
I commented that in the last commit. 
also please check the below : 
https://github.com/Samsung/TizenFX/issues/2329

at this stage my suggestion is to comment that in the guide:

Please be aware, that because of a typo in the source code, both in the code and in the guide, we use the name 'frustrum' instead of 'frustum'.